### ISSUE TITLE
@rematch/loading not working   expected 

### ISSUE THREAD

**Issue by skyblue** [rematch/rematch#930]
**Describe the bug**

dispatch.model.doAsync()       **√**

dispatch({ type: 'mode/doAsync'})      **X**

**Comment by semoal** [comment]
Hi mate.
Can you try to modify our node_modules to this changes and share with us if it's working properly?

https://github.com/rematch/rematch/pull/745/files

Probably this is a bug that we missed on the v2 refactor :)

**Comment by semoal** [comment]
Just tested and works fine. Probably you're using v1 version which is not maintained anymore. You should check our migration guide for v2 ( which is practically anything) and you will get tons of new features, fixes and a considerable bundle-size improvement

**Comment by skyblue** [comment]
```
 "name": "@rematch/core",
  "version": "2.1.0",
```
can' find the src in the version

**Comment by semoal** [comment]
I've created a branch where you can fork and commit there a possible fix, it should pass the testing suite when it's fixed https://github.com/rematch/rematch/commit/46baddfb54f2d935c9b4fe65db3a82789042b296

**Comment by yndue736** [comment]
Yes you can click on the link given in the replies to get the best information about it just like [management assignment helper](https://managementassignmenthelper.com/) I took the same guide from here.
